config/fxdata/magic.cfg

```diff
@@ -969,7 +970,7 @@ PointerSprites = 8
 PanelTabIndex = 10
 SoundSamples = 829
 SoundPlayed = 37
-Power = 80 120 160 210 260 360 480  680  950
+Power = 80 120 160 210 280 360 480  680  950
 Cost = 300 400 500 600 700 800 900 1000 1200
```

Comment:
![image](https://user-images.githubusercontent.com/68837817/123856648-045d2880-d92a-11eb-8af3-9b937eef03a7.png)

Reply:
My idea was that price per hp is now a straight line when plotted. What are you trying to say?

Reply:
I was looking at data. Ok.